packages/expo-updates/ios/EXUpdates/Update/EXUpdatesNewUpdate.m

```diff
@@ -12,13 +12,16 @@
 @implementation EXUpdatesNewUpdate
 
 + (EXUpdatesUpdate *)updateWithNewManifest:(EXManifestsNewManifest *)manifest
-                                  response:(nullable NSURLResponse *)response
+                                   headers:(NSDictionary *)headersDictionary
```

Comment:
```suggestion
                                   headers:(NSDictionary *)headers
```
probably not a big deal, but I would keep the parameter names the same in the header and implementation files